src/main/webapp/app/entities/outcome/outcome.component.html

```diff
@@ -1,74 +1,26 @@
-                    <th scope="col"  jhiSortBy="name"><span jhiTranslate="supportivecareApp.outcome.name">Name</span> <fa-icon icon="sort"></fa-icon></th>
-                    <th scope="col"  jhiSortBy="description"><span jhiTranslate="supportivecareApp.outcome.description">Description</span> <fa-icon icon="sort"></fa-icon></th>
-                    <th scope="col"></th>
-                </tr>
-            </thead>
+        <table class="table table-striped border-style" aria-describedby="page-heading">
             <tbody infinite-scroll (scrolled)="loadPage(page + 1)" [infiniteScrollDisabled]="page >= links['last']" [infiniteScrollDistance]="0">
-                <tr *ngFor="let outcome of outcomes; trackBy: trackId" data-cy="entityTable">
-                    <td><a [routerLink]="['/outcome', outcome.id, 'view']">{{ outcome.id }}</a></td>
-                    <td>{{ outcome.name }}</td>
-                    <td>{{ outcome.description }}</td>
-                    <td class="text-right">
-                        <div class="btn-group">
-                            <button type="submit"
-                                    [routerLink]="['/outcome', outcome.id, 'view']"
-                                    class="btn btn-info btn-sm"
-                                    data-cy="entityDetailsButton">
-                                <fa-icon icon="eye"></fa-icon>
-                                <span class="d-none d-md-inline" jhiTranslate="entity.action.view">View</span>
-                            </button>
-
-                            <button *customHasAnyAuthority="'ROLE_USER'"
-                                    type="submit"
-                                    [routerLink]="['/outcome', outcome.id, 'edit']"
-                                    class="btn btn-primary btn-sm"
-                                    data-cy="entityEditButton">
-                                <fa-icon icon="pencil-alt"></fa-icon>
-                                <span class="d-none d-md-inline" jhiTranslate="entity.action.edit">Edit</span>
-                            </button>
-
-                            <button *customHasAnyAuthority="'ROLE_USER'"
-                                    type="submit" (click)="delete(outcome)"
-                                    class="btn btn-danger btn-sm"
-                                    data-cy="entityDeleteButton">
-                                <fa-icon icon="times"></fa-icon>
-                                <span class="d-none d-md-inline" jhiTranslate="entity.action.delete">Delete</span>
-                            </button>
-                        </div>
-                    </td>
-                </tr>
+            <tr *ngFor="let outcome of outcomes; trackBy: trackId" data-cy="entityTable">
+                <td class= "table-item" [routerLink]="['/drug', outcome.id, 'view']">
+                    <a data-cy="entityDetailsButton" [routerLink]="['/drug', outcome.id, 'view']">
```

Comment:
Should be <a data-cy="entityDetailsButton" [routerLink]="['/outcome', outcome.id, 'view']">